site/content/entry/yuzu-progress-report-dec-2021/index.md

```diff
@@ -0,0 +1,376 @@
+
+{{< single-title-imgs
+    " "
+    "./vidbug.png"
+    "./vidfix.png"
+    >}}
+
+This is a complex issue and the main reason Vulkan is not the default API in use.
+Users of old laptops with AMD and Intel integrated GPUs tend to use the driver shipped by either the laptop vendor or Windows Update. 
+In both cases those drivers are most likely years old (yuzu can run on AMD GPUs from 2012) and either lack Vulkan support at all, or only support a portion of what’s 
+needed to run yuzu.
+Also, since laptops by default connect the display directly to the integrated GPU, that’s the first Vulkan driver that will be seen, so it’s critical to have the latest 
+GPU driver installed *even if* your laptop has a dedicated NVIDIA GPU running the latest driver.
+
+While telling AMD users to [manually download and install updated drivers](https://www.amd.com/en/support) is a viable option and works as it should, in its 
+*infinite wisdom*, Intel decided to block manual installation of its own official drivers if a custom laptop vendor driver is in use (those modified drivers are usually 
+created to cheat on battery life metrics and/or to save money on cooling).
+
+The only alternative in those cases is to [manually download the ZIP version](https://www.intel.com/content/www/us/en/download/19344/intel-graphics-windows-dch-drivers.html) 
+of the driver, unpack it, go to Device Manager, right-click the correct GPU in Display Adapters, select `Update Driver Software…`, `Browse my computer for driver software`, 
+`Let me pick from a list of device drivers on my computer`, `Have Disk…`, then finally browse to the folder where the driver was unpacked and select the `iigd_dch.inf` file.
+What a very intuitive and user-friendly way to update a GPU driver, great job Intel…
+
+With this *easy job* done, the Intel GPU gets full Vulkan support, runs at its intended performance, and has access to all the new features, fixes, and performance 
+improvements that the driver developers worked on. 
+The driver is also allowed to auto-update on new official releases.
+
+Known software that uses broken Vulkan injectors are outdated screen recorders like Bandicam, Action!, and even OBS.
+We strongly recommend using an up to date OBS, the native encoders from the GPU vendor (Radeon ReLive and Geforce Experience), or the integrated Xbox Game Bar on Windows.
+Overwolf is also known to break Vulkan support, so we **strongly** recommend avoiding it.
+
+## Future projects
+
+If you want a sneak peak at the progress on Ring Fit Adventure, here’s a video for you:
+
+{{< imgs
+	"./ringfit.mp4| Ring Fit Adventure, again"
+  >}}
+
+`Project Gaia`, our Virtual File System rewrite is progressing smoothly. 
+As a tip, SSD users will benefit from extra access time performance with it.
+
+Blinkhawk informs us that `Project Y.F.C.` has dissolved into smaller features to get more progressive updates instead of delaying for a big release that would require 
```

Comment:
informs us that YFC will be grouping together the changes into smaller pieces, to get more progressive...